modules/installation-identify-supported-instance-types.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
```

Comment:
Consider adding "aws-outposts" to the file name, as it relevant only for this case